ecl/dns/v2/_proxy.py

```diff
@@ -124,13 +124,20 @@ def get_name_server(self, zone_id):
         """
         return list(self._list(name_server.NameServer, zone_id=zone_id))
 
-    def recordsets(self, zone_id):
+    def recordsets(self, zone_id, limit, marker):
```

Comment:
後方互換性を考慮して以下のようにデフォルト引数値を設定した方が良いと思います。古いcliからlimit, markerを渡されなかった場合にエラーになることを防ぐことができます。
```suggestion
    def recordsets(self, zone_id, limit=None, marker=None):
```

Reply:
That point was not considered.I will fix it.